smm9509

smm9509 opened the pull request:
assignment submission

smm9509 committed:
copy template to new file for my page.

smm9509 committed:
completed blog assignment

smm9509 committed:
changes to better fit the rubric

MoralCode commented on _posts/2025-04-04-smm9509-krita-453572.md:
```diff
@@ -0,0 +1,121 @@
+
+After that consideration, I searched and browsed the [bug tracker](https://bugs.kde.org/).
+
+![searching for krita issues in the bug tracker](.attachments.4761187/image%20%282%29.png)
+
+After browsing the bugs, I found one I liked.
+
+![image (3).png](.attachments.4761187/image%20%283%29.png)
+
+I picked bug [453572](https://bugs.kde.org/show_bug.cgi?id=453572), that I felt would be easy and valuable to solve.
+
+The bug 453572, titled "JPEG-XL advanced export options confusing" complains about vocabulary being too technical so that it's difficult to understand and localize.
+
+![screenshot of bug report](.attachments.4761187/image.png)
```
These images dont appear to be showing up on my end - did you commit them to your branch?

MoralCode reviewed (changes requested):
This looks like a very detailed blog post, nice job!

The only issue i see is that the images aren't showing up - can you add those (take a look at some of the other merged PR's here if you want examples of where to put them in the repo or how to reference them in the markdown)

smm9509 committed:
I think I fixed the embedded images issue but I'm not sure because
the course documentation (readme, assignment description, etc) lacks build instructions (
to convert the .md source files to a viewable webpage)

smm9509 commented:
@MoralCode, when you said "images aren't showing up", how did you test whether the images show up or not? what's the build process?

MoralCode commented:
> @MoralCode, when you said "images aren't showing up", how did you test whether the images show up or not? what's the build process?

i am cloning/checking out your branch and then running the dev environment script in the root of the repo (which has jekyll and all its dependencies in a containerized environment because ruby is kind of a pain to work with natively). This will serve a local copy of the site on localhost that i can use to visually check if everything loaded right

MoralCode commented:
I do in fact see your three images now, so youre all set!
